controllers/workplacement_controller.go

```diff
@@ -51,7 +51,10 @@ type WorkPlacementReconciler struct {
 	VersionCache map[string]string
 }
 
-const repoCleanupWorkPlacementFinalizer = "finalizers.workplacement.kratix.io/repo-cleanup"
+const (
+	repoCleanupWorkPlacementFinalizer       = "finalizers.workplacement.kratix.io/repo-cleanup"
+	kratixFileCleanupWorkPlacementFinalizer = "finalizers.workplacement.kratix.io/kratix-file-cleanup"
```

Comment:
this is a bit generic when compared to the one above, maybe `filepath-mode-none-cleanup` or `kratix-dot-file-cleanup`, or something more descriptive?